pkg/scheduler/scheduler.go

```diff
@@ -1,52 +0,0 @@
-package scheduler
-
-import (
-	"log"
-	"runtime/debug"
-
-	"github.com/robfig/cron/v3"
```

Comment:
shouldn't there be module changes in `go.mod` after we remove this pkg?

Reply:
Thank you. missed updating this. updated now.